package.json

```diff
@@ -46,12 +46,14 @@
     ]
   },
   "dependencies": {
+    "@types/react-table": "7.7.9",
```

Comment:
`@types` gaan meestal bij de devDependencies tenzij dit voor een reden gedaan is?